prog-o-meter.py

```diff
@@ -146,7 +152,39 @@ def add_day(self):
         update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
         if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
+            self.add_day_button.config(state = "disabled")  
+    def log_entry(self):
+        """Opens a new window for user to make a new log entry. The user can make any number of entries he/she wishes.

```

Comment:
I have used "they" instead of he/she other places in the documentation, so for consistency, maybe we can do that?

Reply:
Changed.